### ISSUE TITLE
Periodically update the ceph-build checkout

### ISSUE THREAD

**Issue by djgalloway** [alfredodeza/prado#22]
It'd be cool if the service automatically kept the ceph-build repo up to date.

**Comment by alfredodeza** [comment]
I don't think that should be up to the service... but we should probably get better deployment automation
